apis/aws.go

```diff
@@ -33,6 +33,7 @@ import (
 	efsv1alpha1 "github.com/crossplane/provider-aws/apis/efs/v1alpha1"
 	eksv1alpha1 "github.com/crossplane/provider-aws/apis/eks/v1alpha1"
 	eksv1beta1 "github.com/crossplane/provider-aws/apis/eks/v1beta1"
+	elasticache1alpha1 "github.com/crossplane/provider-aws/apis/elasticache/v1alpha1"
```

Comment:
@namku due to legacy naming patterns, we currently have elasticache types in the `cache` group. @muvaf do you think we should continue with the old naming pattern?

Reply:
I think it make sense to use the same serviceid name as in the list (https://github.com/aws/aws-sdk-go/tree/v1.34.32/models/apis), I think is better solution to change the others resources from cache to elasticache. What do you think? @hasheddan @muvaf 
I can change to cache serviceid name if you think is better solution keep this serviceid name.

Reply:
The code generator decides on the [folder name here](https://github.com/aws-controllers-k8s/code-generator/blob/907b085/cmd/ack-generate/command/crossplane.go#L56), which would be whatever AWS decided to use as service name.

Reply:
I would agree we need to be consistent under `elasticache` which would mean moving existing ones under `cache`. But, this would be an API change assuming we would also want to change the group name at CRD level (e.g. `apiVersion`). 
Maybe we can just create a follow-up issue for that and handle with a separate PR.

Reply:
we will open new PR to change cache to elasticache but it is breaking change as mentioned